pkg/scripts/poststart/check_fuse.go

```diff
@@ -61,9 +62,9 @@ echo "succeed in checking mount point $ConditionPathIsMountPoint"
 	contentUnprivilegedSidecar = `#!/bin/bash
 set -ex
 
-echo "Sending deivce ioctl to /dev/fuse"
-chmod u+x /tools/ioctl_sync
+echo "Sending device ioctl to /dev/fuse"
 /tools/ioctl_sync
```

Comment:
How about using `timeout /tools/ioctl_sync`, and determine the return value to check if it's timeout.